src/app/sb-admin/service/user.service.ts

```diff
@@ -40,4 +40,17 @@ export class UserService {
     })
     return this.http.post(environment.userCreateUrl, payload, { headers: header });
   }
+
+  blockUnblockUser(payload: any, userStatus: any): Observable<any> {
+    let header = new HttpHeaders({
+      "Content-Type": 'application/json',
+      "Authorization": environment.authKey,
+      "x-authenticated-user-token": environment.userToken,
+    })
+    if (userStatus) {
+      return this.http.post(environment.blockUserUrl, payload, { headers: header });
```

Comment:
Is block and unblock have different urls?

Reply:
yes sir

Reply:
These URLs should come from url configs json. I think this is resolved in another PR.